spec/cf_app_discovery/app_info_configurer_spec.rb

```diff
@@ -12,15 +12,26 @@ def get(path)
           resources: [{
             entity: {
               domain_url: "sample_domain_url",
-              host: "sample_host"
+              host: "sample_host",
+              path: ""
+            }
+          }]
+        },
+        "/v2/apps/sample_guid_with_host_and_path/routes" => {
+          resources: [{
+            entity: {
+              domain_url: "sample_domain_url",
+              host: "sample_host",
+              path: "/sample_path"
             }
           }]
         },
         "/v2/apps/sample_guid_without_host/routes" => {
```

Comment:
Could there be the case where we have no host but do have a path that we should test?

Reply:
yes there could. i'll add a test